templates/core/terraform/airlock/locals.tf

```diff
@@ -33,4 +33,5 @@ locals {
   import_rejected_eventgrid_subscription_name   = "evgs-airlock-import-rejected-blob-created"
   export_approved_eventgrid_subscription_name   = "evgs-airlock-export-approved-blob-created"
 
+  airlock_function_app_name = "funcal${var.tre_id}"
```

Comment:
```suggestion
  airlock_function_app_name = "func-airlock-${var.tre_id}"
```

Reply:
When you create an Azure function using the portal it allows to add hyphens to the name. but when it is done using Terraform it says hyphens are illegal characters...
Another thing, using Terraform the name length should be between 3 to 24, so I need to keep it short to accommodate for a long tre id.

Reply:
even so, I suggest the name has "air" or "airlock" in it as a higher priority to "func"

Reply:
just `airlock` would or `airlockp` (airlock processor)

(the fact it's a function isn't so important)

Reply:
I wanted to keep the known abbreviation prefix
Im good with airlockp

Reply:
false alarm. I also used the name for the storage account....